brage-import/src/test/java/no/sikt/nva/brage/migration/lambda/BrageEntryEventConsumerTest.java

```diff
@@ -1197,42 +1264,44 @@ void whenCreatingNewPublicationTheHandleReportShouldBeStoredUnderHandleReportsFo
         assertThat(storedHandleString, is(not(nullValue())));
     }
 
-    @Test
-    void shouldMergeCristinPublicationEvenWhenMostDataIsMissing()
-        throws BadRequestException, IOException, nva.commons.apigateway.exceptions.NotFoundException {
-        var cristinIdentifier = randomString();
-        var cristinPublication = createCristinPublication(cristinIdentifier);
-        var minimalRecord = createMinimalRecord(cristinIdentifier);
-        var contentFile = createContentFile();
-        minimalRecord.setContentBundle(new ResourceContent(List.of(contentFile)));
-        var s3Event = createNewBrageRecordEvent(minimalRecord);
-        handler.handleRequest(s3Event, CONTEXT);
-        var updatedPublication = resourceService.getPublication(cristinPublication);
-
-        //assert that dummy handles has not been stored in the updated publication
-        assertThat(updatedPublication.getHandle(), not(equalTo(minimalRecord.getId())));
-        assertThat(updatedPublication.getAdditionalIdentifiers(), not(hasItem(
-            new AdditionalIdentifier("handle",
-                                     minimalRecord.getId().toString()))));
-        var associatedArtifacts = updatedPublication.getAssociatedArtifacts();
-
-        // assert that  contentFile was copied
-        assertThat(associatedArtifacts, hasSize(1));
-        var publishedFile = (PublishedFile) associatedArtifacts.getFirst();
-        assertThat(publishedFile.getName(), is(equalTo(contentFile.getFilename())));
-        assertThat(publishedFile.getIdentifier(), is(equalTo(contentFile.getIdentifier())));
-        assertThat(publishedFile.getLicense(), is(equalTo(contentFile.getLicense().getNvaLicense().getLicense())));
-        assertThat(publishedFile.getPublisherVersion(), is(equalTo(minimalRecord.getPublisherAuthority().getNva())));
-
-        //assert that we are storing reports based on the dummy handles:
-        var updateHandleReporstFolder = UnixPath.of(UPDATE_REPORTS_PATH);
-        var filesInUpdatedHandleReportsFolder = s3Driver.getFiles(updateHandleReporstFolder);
-        assertThat(filesInUpdatedHandleReportsFolder, is(not(empty())));
-        var storedHandleString = extractUpdateReportFromS3ByUpdateSource(s3Event,
-                                                                         cristinPublication,
-                                                                         minimalRecord.getId(), "CRISTIN");
-        assertThat(storedHandleString, is(not(nullValue())));
-    }
+//TODO: I guess this test is for import of Unis. We have to identify unis records, otherwise they will not be merged.
+
+//    @Test
+//    void shouldMergeCristinPublicationEvenWhenMostDataIsMissing()
```

Comment:
I guess this is Unis publication we are testing here? We have to find other requirements for Unis, as for now publication without contributors/publicationDate.year and so on will never be merged.